napari/components/add_layers_mixin.py

```diff
@@ -318,6 +342,14 @@ def add_points(
             Scale factors for the layer.
         translate : tuple of float
             Translation values for the layer.
+        rotate : float, 3-tuple of float, or n-D array.
+            If a float convert into a 2D rotate matrix using that value as an
+            angle. If 3-tuple convert into a 3D rotate matrix, rolling a yaw,
+            pitch, roll convention. Otherwise assume an nD rotate. Angle
+            conversion are done either using degrees or radians depending on the
+            degrees boolean parameter.
```

Comment:
Same question as above: where is the degrees boolean parameter and how do I use it? We should add this to the docstring.

Reply:
See response above